resources/hpxml-measures/.rubocop.yml

```diff
@@ -48,7 +48,8 @@ Style/DocumentationMethod:
 Style/FrozenStringLiteralComment:
   Enabled: true
 Style/HashSyntax:
-  Enabled: true
+  Enabled: false
+  EnforcedShorthandSyntax: never
```

Comment:
Bugfix to prevent rubocop from updating a huge number of measures. This change was already made in OS-HPXML so this will merge cleanly once the latest OS-HPXML is pulled in.